data/xml/2022.eamt.xml

```diff
@@ -0,0 +1,881 @@
+      <pages>13–22</pages>
+      <abstract>Building effective Neural Machine Translation models often implies accommodating diverse sets of heterogeneous data so as to optimize performance for the domain(s) of interest. Such multi-source / multi-domain adaptation problems are typically approached through instance selection or reweighting strategies, based on a static assessment of the relevance of training instances with respect to the task at hand. In this paper, we study dynamic data selection strategies that are able to automatically re-evaluate the usefulness of data samples and to evolve a data selection policy in the course of training. Based on the results of multiple experiments, we show that such methods constitute a generic framework to automatically and effectively handle a variety of real-world situations, from multi-source domain adaptation to multi-domain learning and unsupervised domain adaptation.</abstract>
+      <url hash="f6c1e291">2022.eamt-1.4</url>
+      <bibkey>pham-etal-2022-multi</bibkey>
+    </paper>
+    <paper id="5">
+      <title>The use of online translators by students not enrolled in a professional translation program: beyond copying and pasting for a professional use</title>
+      <author><first>Rudy</first><last>Loock</last></author>
+      <author><first>Sophie</first><last>Léchauguette</last></author>
+      <author><first>Benjamin</first><last>Holt</last></author>
+      <pages>23–29</pages>
+      <abstract>In this paper, we discuss a use of machine translation (MT) that has been quite overlooked up to now, namely by students not enrolled in a professional translation program. A number of studies have reported massive use of free online translators (OTs), and it seems important to uncover such users’ abilities and difficulties when using MT output, whether to improve their understanding, writing, or translation skills. We report here a study on students enrolled in a French ‘applied languages program’ (where students study two languages, as well as law, economics, and management). The aim was to uncover how they use OTs, as well as their (in)ability to identify and correct MT errors. Obtained through two online surveys and several tests conducted with students from 2020 to 2022, our results show an unsurprising widespread use of OTs for many different tasks, but also some specific difficulties in identifying MT errors, in particular in relation to target language fluency.</abstract>
+      <url hash="d735fdf8">2022.eamt-1.5</url>
+      <bibkey>loock-etal-2022-use</bibkey>
+    </paper>
+    <paper id="6">
+      <title>Comparing and combining tagging with different decoding algorithms for back-translation in <fixed-case>NMT</fixed-case>: learnings from a low resource scenario</title>
+      <author><first>Xabier</first><last>Soto</last></author>
+      <author><first>Olatz Perez-De-Vi
+
+textasciitilde</first><last>Naspre</last></author>
+      <author><first>Gorka</first><last>Labaka</last></author>
```

Comment:
This is obviously mis-parsed (should be Perez-De-Viñaspre). @xinru1414 what does the source look like? Can you fix this in the ingestion script? Same for `\"{i}`.